packages/front-end/pages/features/[fid].tsx

```diff
@@ -489,7 +489,10 @@ export default function FeaturePage() {
               count={rules.length}
               padding={false}
             >
-              <div className="appbox mb-4 border-top-0">
+              <div
```

Comment:
Accidentally left this in, but it needs to go out anyhow to fix https://github.com/growthbook/growthbook/issues/1003. Hopefully ok to leave it in.